cpp/ql/src/semmle/code/cpp/ir/implementation/aliased_ssa/Instruction.qll

```diff
@@ -804,12 +804,26 @@ class CopyValueInstruction extends CopyInstruction, UnaryInstruction {
   final override UnaryOperand getSourceValueOperand() { result = getAnOperand() }
 }
 
+/**
```

Comment:
All these diffs in all copies of `Instruction.qll` should go away once #4502 is merged.